composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "google/apiclient",
+    "name": "pdobrovolny/apiclient",
```

Comment:
🤔

Reply:
oh sorry, my bad ... That belong to my fork. Apology

Reply:
by the way, the code seems to be compatible with monolog version 3. Perhaps You could fix those dependecies.